src/table/cells/withoutSelections.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
I dont think this is a good idea. There will be another HOC soon for column styling, so we should only have `with...` HOCs. It should still work if you set `className` in `tableBodyWrapper` and then return just `TableCell` from `getCellRenderer`, the way it was before

Reply:
yeah, sure

Reply:
Done